### PR TITLE
Create deploy.yml

### PR DESCRIPTION
**Pull request by WildBunnie**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Deployment
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```
Comment (pdPinho) [inline .github/workflows/deploy.yml:7]: can you not remove this?

**Review by pdPinho — APPROVED** [review]
Other than that, it seems fine to me

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (WildBunnie) [reply .github/workflows/deploy.yml:7]: it is used to call the workflow manually

**Review by fungame2270 — APPROVED** [review]
lgtm

**Pull request merged**